### ISSUE TITLE
deeplab pascal finetune on trainval set parameters

### ISSUE THREAD

**Issue by kaixinxiao** [tensorflow/models#6377]
System information

    What is the top-level directory of the model you are using: deeplab
    Have I written custom code (as opposed to using a stock example script provided in    TensorFlow): no
    OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 16.04
    TensorFlow installed from (source or binary): binary
    TensorFlow version (use command below): 1.10
    Bazel version (if compiling from source): no
    CUDA/cuDNN version: 9.0
    GPU model and memory: TeslaV100(32G) 
    Exact command to reproduce: no

Describe the problem
I don't pretrain on COCO. I use bn=16,OS=16 and atrous rate[6,12,18] on train_aug dataset.
When I finetune on trainval set, I use bn=16 ,freeze bn, atrous rate[12,24,36] ，OS=8 and learning rate=0.0001.
But result is not good I think. Are these parameters true，especially in finetune? I also find the paper says learning rate is 0.001 in finetune but the code uses 0.0001 .

**Comment by northeastsquare** [comment]
1.What do you mean by OS=16?
2.Have you used backbone pretrain model on imagenet?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
